reference-adapter/ifix-department-entity-service/src/main/java/org/egov/web/models/persist/DepartmentHierarchyLevel.java

```diff
@@ -0,0 +1,37 @@
+package org.egov.web.models.persist;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Size;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+//@Table(name = "department_hierarchy_level")
+public class DepartmentHierarchyLevel {
+    private String id = null;
+
+    @Size(max = 64)
+    private String tenantId = null;
+
+    @Size(max = 64)
+    private String departmentId = null;
+
+    private String label = null;
+
+    @Size(max = 64)
+    private String parent = null;
+
+    private Integer level = null;
+
+    private String createdBy = null;
```

Comment:
Instead of having these separate properties, create an AuditDetails object. Persister can handle it. 
If we do that, then DTO and this object will become the same, so we can delete the DTO object and reuse this in its place. 
This will also simplify the code a lot, because many LoC are around copying attributes from DTO to this object.

Reply:
It is being used for getting value from db table. It is not being involed in persister flow.
It is just entity corresponding to db table. Just like we do in JPA model. 
It is requied to perceive data while fetching data from DB table.